docs/user/quick-start-browser.md

```diff
@@ -0,0 +1,226 @@
+# Grafana Javascript Agent quickstart (browser)
+
+This document describes basic setup and usage. Also see [demo application](https://github.com/grafana/grafana-javascript-agent/tree/main/demo).
+
+## Set up Grafana Agent
+
+First, a Grafana Agent instance has to be set up, see [Grafana Agent set up documentation](https://grafana.com/docs/agent/latest/set-up/).
+It has to be configured with `app-agent-receiver` integration to expose a http collection
+endpoint and run with `integrations-next` flag enabled.
+
+Below is an example of a basic Grafana Agent config that will expose a collector endpoint
+at [http://host:12345/collect](http://host:12345/collect), and forward collected telemetry to Loki,
+Tempo and Prometheus instances. See [agent app receiver integration documentation](https://github.com/grafana/agent/blob/main/docs/user/configuration/integrations/integrations-next/app-agent-receiver-config.md)
+for more details on available options.
+
+```yaml
+metrics:
+  wal_directory: /tmp/wal
+  global: {}
+  configs:
+    - name: default
+      remote_write:
+        - url: https://prometheus-us-central1.grafana.net/api/prom/push
+          basic_auth:
+            username: xxx
+            password: xxx
+logs:
+  positions_directory: /tmp/loki-pos
+  configs:
+    - name: default
+      scrape_configs: []
+      clients:
+        - url: https://xxx:xxx@logs-prod-us-central1.grafana.net/loki/api/v1/push
+traces:
+  configs:
+    - name: default
+      remote_write:
+        - endpoint: tempo-us-central1.grafana.net:443
+          basic_auth:
+            username: xxx
+            password: xxx
+      receivers:
+        otlp:
+          protocols:
+            grpc:
+            http:
+              cors:
+                allowed_origins:
+                  - http://localhost:1234
+                max_age: 7200
+integrations:
+  app_agent_receiver_configs:
+    - autoscrape:
+        enable: true
+        metrics_instance: "default"
+      api_key: "secret" # optional, if set, client will be required to provide it via x-api-key header
+      instance: "frontend"
+      logs_instance: "default"
+      traces_instance: "default"
+      server:
+        host: 0.0.0.0
+        port: 12345
+        cors_allowed_origins:
+          - "https://my-app.example.com"
+      logs_labels: # labels to add to loki log record
+        app: frontend # static value
+        kind:         # value will be taken from log items. exception, log, measurement, etc
+      logs_send_timeout: 5000
+      sourcemaps:
+        download: true # will download source file, extract source map location,
+        # download source map and use it to transform stack trace locations
+```
+
+## Install Grafana Javascript Agent
+
+```bash
+#npm
+npm i -S @grafana/agent-web
+
+#yarn
+yarn add @grafana/agent-web
+```
```

Comment:
What are the steps for installing? Can you describe in 1.,2.3. and so on, so the user knows what to do? Or maybe it's as simple as one step "1. Enter the following in your command line." Maybe also add a result - what happens after? What are the next steps?

Reply:
Will try to split this up in discreet steps.

I'd rather avoid detailed/verbose instructions for installing dependencies and aim to simply communicate that this particular dependency is required. Based on my experience, giving `npm` and/or `yarn` commands is the usual way of doing it. I think we can assume reader will be competent on basic web app development topics :)

Reply:
Ok--but let's include at least an introductory sentence here like To install Grafana Javascript Agent, run the following: